packages/http-client-java/generator/http-client-generator-clientcore-test/Generate.ps1

```diff
@@ -118,13 +118,13 @@ try {
   }
 
   # smoke test, generate Java project and verify compilation pass
-  git fetch origin pull/6981/head:smoke-test-branch
-  git restore --source smoke-test-branch --worktree -- ../../../smoke-http-specs
-  Copy-Item -Path ../../../smoke-http-specs/specs -Destination ./ -Recurse -Force
-  Generate-Compile todoapp
-  Generate-Compile petstore
-  Remove-Item ./specs -Recurse -Force
-  Remove-Item ../../../smoke-http-specs -Recurse -Force
+#   git fetch origin pull/6981/head:smoke-test-branch
```

Comment:
This too, any reason to remove?

Reply:
The script here uses code from a PR head. It may not work in local (e.g. if the remote is not `origin`).

One can disable it, and the same tsp would be included in http-specs in its next release.
Or one can for now use the tsp in this repo https://github.com/microsoft/typespec/tree/main/packages/http-specs/smoke

The purpose of this smoke test is to generate a package, run `mvn package` (the line above in `Generate-Compile` func) on it to make sure it builds pass (that should include the triggering of annotation processor from the generated pom in package).